core/server/data/meta/amp_url.js

```diff
@@ -1,9 +1,9 @@
-var urlService = require('../../services/url'),
+const urlService = require('../../services/url'),
     getUrl = require('./url'),
     _ = require('lodash');
 
 function getAmplUrl(data) {
-    var context = data.context ? data.context : null;
+    let context = data.context ? data.context : null;
```

Comment:
Could you please PR the ES6 changes in a separate PR?
The PR is hard to review now.

NOTE: We use constantly const in Ghost,

Reply:
Reverted for files that I did not change, is that ok, or do I do this for all?

Reply:
Looks much better, thanks 👍